source/js/buyers-guide/components/primary-nav/primary-nav.scss

```diff
@@ -1,6 +1,3 @@
-$pni-primary-nav-breakpoint: $bp-sm;
-$pni-primary-nav-container-z-index: $zindex-sticky;
```

Comment:
These have been moved to [source/sass/buyers-guide/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-e5e5421422d79fcbdb311aed60c81209c8785ca2f292dba3d6dbab4b492030f4)